script/prepare-llvm-linux.sh

```diff
@@ -28,6 +28,8 @@ $CP $ZLIB/lib/libz.so* stage1/lib/
 find stage1 -type f -exec strip --strip-unneeded '{}' \; 2> /dev/null
 # lean.h dependencies
 $CP llvm/lib/clang/*/include/{std*,__std*,limits}.h stage1/include/clang
+# LLVM backend includes
+$CP -r llvm/include/* stage1/include/
```

Comment:
Why do we need to ship this?

Reply:
That is true, but I'm not sure if that warrants shipping 21MB of headers (counting only `llvm/`) to everyone!

Reply:
I've removed copying the `include`/ subdirectory in the latest version.